client/components/App/App.jsx

```diff
@@ -39,8 +43,10 @@ function App(props) {
 
 App.propTypes = {
   isOrdersVisible: PropTypes.bool.isRequired,
+  isAuthenticated: PropTypes.bool.isRequired,
   onMount: PropTypes.func.isRequired,
   onCategoriesLoad: PropTypes.func.isRequired,
+  onUserChange: PropTypes.func.isRequired,
   categories: PropTypes.arrayOf(PropTypes.any.isRequired).isRequired,
```

Comment:
@IgorChaikin `PropTypes.arrayOf(PropTypes.any.isRequired)` means, that is just array of anything, it is not very helpful, especially when structure of `category` item is known. Could you search `PropTypes.arrayOf(PropTypes.any.isRequired).isRequired` in project and replace detailed definition, BTW you could move entities definition to some file, and import them and reuse, to not repeat yourself